com.unity.render-pipelines.core/Runtime/Debugging/VolumeDebugSettings.cs

```diff
@@ -154,18 +120,26 @@ public static string ComponentDisplayName(Type component)
             return component.Name;
         }
 
-        internal static List<HDAdditionalCameraData> cameras { get; private set; } = new List<HDAdditionalCameraData>();
+        protected static List<T> additionalCameraDatas { get; private set; } = new List<T>();
 
-        internal static void RegisterCamera(HDAdditionalCameraData camera)
+        /// <summary>
+        /// Register the camera for the Volume Debug
+        /// </summary>
+        /// <param name="additionalCamera">The camera with it's additional camera data</param>
```

Comment:
`it's` -> `its`, or maybe something like `The AdditionalCameraData of the camera to be registered.`. If you change it, do the same for `UnRegisterCamera`.